src/equation_of_state.jl

```diff
@@ -1,12 +1,10 @@
-#=
+"""
+    LinearEquationOfState(; kwargs...)
```

Comment:
Hmmm, will Documenter.jl know how to expand this?

If not, might be good to have a docstring that will explain all the keyword arguments. I wonder if `Base.@kwdef` helps with that.

Reply:
No, I don't think it knows. 

It'd be good to know what is recommended for documenting functions with long lists of keyword arguments.

Reply:
Yeah we can figure it out when we improve the docs.